source/bao_hyp/user_manual/porting.rst

```diff
@@ -0,0 +1,270 @@
+Porting
+=======
+
+The Bao Hypervisor only contains architecture-specific drivers (e.g., interrupt controllers or
+IOMMUS). Therefore porting the hypervisor essentially resumes to two simple operation: creating a
+platform description folder containing a platform definition source and header files which
+indicates, among others, the number of CPUs or the memory available. The exception is an UART driver
+used fore logging, so if a driver is not available for a UART on the target platform this might
+require writing such a driver. Finally, given for some platforms provide a custom mechanism to
+define DMA device Master IDs required for configuring IOMMUs, one might require to configure
+firmware to set these IDs. In sum, porting Bao to a platform requires:
+
+1) Create a platform directory containing a platform description [mandatory];
+2) Writing an UART driver [if the driver for the target UART is not available];
+3) Setting up firmware to define DMA device Master IDs used by the platform IOMMU [platform dependent];
```

Comment:
Here it would be useful to have a link to each section that describes the process.